app/api/v1/airline/[airlineId]/route.ts

```diff
@@ -0,0 +1,316 @@
+import { NextRequest, NextResponse } from "next/server"
+
+import { getDatabase } from "@/lib/couchbase-connection"
+import { TAirline,AirlineSchema } from "@/app/models/Airline"
+import { DocumentNotFoundError,DocumentExistsError } from 'couchbase';
+
+import { ZodError } from "zod";
+
+/**
+ * @swagger
```

Comment:
Why do i get No links on the right hand side of the swagger response on the UI?
Can you pls remove it?
<img width="1289" alt="image" src="https://github.com/couchbase-examples/nextjs-capella-quickstart/assets/108796209/236a1f6e-23e7-424b-bd8c-07afe92cb6a0">

Reply:
There is no currently no way to remove the "no links" from the nextjs swagger package.We can only add links